docs/use-cases/NVIDIA-GPU-passthrough-and-Kata.md

```diff
@@ -0,0 +1,298 @@
+# Using NVIDIA GPU device with Kata Containers
```

Comment:
ooi, when did the capitalisation change, or has it always been like that officially? It looks like [wikipedia needs to be updated on this point](https://en.wikipedia.org/wiki/Nvidia) and [and so does our spell checker](https://github.com/kata-containers/tests/blob/main/cmd/check-spelling/data/main.txt#L72) :smile:

Reply:
Spell check update: https://github.com/kata-containers/tests/pull/4605 
Wikipedia update: https://en.wikipedia.org/wiki/Nvidia